### ISSUE TITLE
Documentation: add a section with videos on Spine tools

### ISSUE THREAD

**Issue by suvayu** [spine-tools/Spine-Toolbox#2254]
Now that the repository description link points to the new webpage, the YouTube links included in the [old website](http://www.spine-model.org/spine_webinar.htm) are virtually unfindable (also on web search, I think because the website is HTTP-only).  We could create a new section at the end of "Getting started".

Thoughts?

**Comment by PekkaSavolainen** [comment]
Getting Started is quite long though so I'm worried that nobody will find them. Maybe a better place for YouTube links would be in the index page of the User Guide and also in the README.md as a new 'tag' after 'chat on gitter'.

**Comment by suvayu** [comment]
Yes, you are correct.  I was also thinking about that while writing our the suggestion.  The index page might not be particularly visible either, but probably a good place for now.  Including in the README would be perfect actually!  I'll submit a PR for the README

**Comment by soininen** [comment]
We could have a Links section as a top-level page on the documentation so it's discoverable from the contents bar. Naturally, relevant sections should link to it, too.

**Comment by suvayu** [comment]
Something like "Other resources for learning", or simply "Resources"?

**Comment by soininen** [comment]
"Resources" sounds too general and might be confused with project item resources so I would go with something more specific.

**Comment by jkiviluo** [comment]
Sorry that I did not spot this earlier. We have videos from Mopo project too (the ones explained now in Readme are from Spine project):
Toolbox: https://www.youtube.com/watch?v=oRwXwV9tuSo
SpineOpt: https://www.youtube.com/watch?v=brf6rlzNDR0
Spine Interface: https://www.youtube.com/watch?v=oiY8yUkQV8c

We should get those in the right channel though. The idea was to use this channel: https://www.youtube.com/@DOESVTT/featured, but the handle should be more generic than DOESVTT.
